### ISSUE TITLE
Update documentation in preparation of release 1.5

### ISSUE THREAD

**Issue by vortex-5** [vortex-5/ddwrt-bwmon#8]
Version 1.5 is basically ready for release. Installation documentation has not yet been updated and needs a go over before I can publically release version 1.5.

**Comment by AleXndrTheGr8st** [comment]
I'm looking forward to version 1.5. You mentioned in a closed issue that changing the sample time from 10s to something else broke calculation. I changed mine to 1s (in both the bwmon-running.sh and the bwmon.js files), and it now updates every second. Running speed tests, I'm not seeing any difference from the stats shown with a 10s sample rate.

**Comment by vortex-5** [comment]
There shouldn't be a huge difference in speed tests however it does mean that your router is working hard even when no one is actively viewing the page. 1.5 uses a php Service that is able to poll on demand so the faster 1 second polling is not used while no one is visiting the page. The router will normally use a longer 60sec stat gather for data consumption measurements and only when you visit the page will you get the faster 1 sec polling. 

For this new version lighttpd on ddwrt is utilized so we can run a service. 

The main advantage is even less load when you're not looking at the active bandwidth. 

The scripts will automatically fall back to the old way of gathering bandwidth if lighttpd isn't available. 

The code is actually done and on the lighttpd branch I just haven't had time to update the readme. MD and other documentation so that's why it hasn't been released yet. I actually have been running 1.5 for a month now to eliminate QA issues.

**Comment by AleXndrTheGr8st** [comment]
Well, I have a pretty powerful router, so even with the poll rate set to 2s, the cpu usage never even tops 50%, thanks to how efficient you've made the scripts. Most of the brunt work is handled by the js, which is a really nice touch and something I never would have thought to do (I was attempting to modify wrtbwmon myself before I stumbled upon your rehaul of it).

**Comment by vortex-5** [comment]
Lol thanks even though I have a pretty powerful router as well my goal is to hit below 1% which is a pretty tight constraint.

Although I do that more so that people with older routers should still be able to run this without adverse effect.

If you want to try the 1.5 branch yourself you are welcome to the main difference is if you enable lighttpd you will need to go to router:81/bwmon.html instead of router/user/bwmon.html

if you don't enable lighttpd the script is capable of automatically falling back to 10s polling so it will work just like the old script.

(Some people on the forums told me they were not getting lighttpd for their router due to lack of flash so I had to design a fallback so they can still receive updates)

I've been running 1.5 for a month myself so far only reason I'm taking so long before full release is this is the first substantial overhaul where the script queries a service the service executes the commands and sends the results back to the script the benefit here is the script can poll arbitrarily fast and it won't put a load on the router until you visit the page. Right now it's set to 2 seconds I think on the lighttpd branch 1s works fine too I went with 2s because visually 1s the page looked like it was on ADHD constantly flashing lol.
